landscaper/utils/helper/helper.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helper
+
+import landscaperv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+
+// GetCondition returns the condition with the given <conditionType> out of the list of <conditions>.
+// In case the required type could not be found, it returns nil.
+func GetCondition(conditions []landscaperv1alpha1.Condition, conditionType landscaperv1alpha1.ConditionType) *landscaperv1alpha1.Condition {
```

Comment:
the landscaper helper function could be used https://github.com/gardener/landscaper/blob/master/apis/core/v1alpha1/helper/helpers.go#L47